searx/utils.py

```diff
@@ -465,3 +464,10 @@ def get_xpath(xpath_str):
 def eval_xpath(element, xpath_str):
     xpath = get_xpath(xpath_str)
     return xpath(element)
+
+
+def get_external_bang_operator():
+    """
+    :return: Returns the external bang operator used in Searx.
+    """
+    return "!!"
```

Comment:
I think this constant ```!!``` can move to query.py. Except if you see some further development, there is no need to declare a function ?

Reply:
I was not entirely sure where to put this but I currently use this in external_bangs.py and in query.py. I find hard coding strings a bad practice. But maybe external_bang.py is a better location for it?

Reply:
After all the refactors the method was not needed anymore so it is removed.